app/src/main/res/layout/search_item.xml

```diff
@@ -1,53 +1,68 @@
-        android:background="?selectableItemBackgroundBorderless"
-        android:padding="@dimen/small_margin"
-        android:src="@drawable/ic_chevron_left_vector" />
+        <com.simplemobiletools.commons.views.MyEditText
+            android:id="@+id/search_query"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/normal_margin"
+            android:layout_marginEnd="@dimen/normal_margin"
+            android:layout_weight="1"
+            android:background="@null"
+            android:hint="@string/search"
+            android:maxLength="50"
+            android:maxLines="1"
+            android:singleLine="true"
+            android:textCursorDrawable="@null" />
 
-    <ImageView
-        android:id="@+id/search_next"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="@dimen/smaller_margin"
-        android:background="?selectableItemBackgroundBorderless"
-        android:padding="@dimen/small_margin"
-        android:src="@drawable/ic_chevron_right_vector" />
+        <ImageView
+            android:id="@+id/search_previous"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/smaller_margin"
+            android:background="?selectableItemBackgroundBorderless"
+            android:padding="@dimen/small_margin"
+            android:src="@drawable/ic_chevron_left_vector" />
 
-    <ImageView
-        android:id="@+id/search_clear"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="@dimen/medium_margin"
-        android:background="?selectableItemBackgroundBorderless"
-        android:padding="@dimen/small_margin"
-        android:src="@drawable/ic_cross_vector" />
-</LinearLayout>
+        <ImageView
+            android:id="@+id/search_next"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/smaller_margin"
+            android:background="?selectableItemBackgroundBorderless"
+            android:padding="@dimen/small_margin"
+            android:src="@drawable/ic_chevron_right_vector" />
+
+        <ImageView
+            android:id="@+id/search_clear"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/medium_margin"
+            android:background="?selectableItemBackgroundBorderless"
+            android:padding="@dimen/small_margin"
+            android:src="@drawable/ic_cross_vector" />
+    </LinearLayout>
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_gravity="bottom"
+        android:alpha="0.1"
+        android:background="#fff" />
```

Comment:
always use 6-8 characters at hex codes, but in this case you can also use color/md_grey_white, it is in commons